cdp4dal/src/main/java/cdp4dal/exceptions/DalReadException.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
The exceptions that we declare here are CDP4 custom exceptions. I notice that you follow the C# pattern to create these exceptions and not the Java pattern. I propose you refactor these to follow the Java pattern. I found multiple articles on the web that cover this topic. This one is very readable (als for Java noob such as myself) https://stackify.com/specify-handle-exceptions-java. Having read this article i think these exceptions should be `checked exception`s and therefore derive from `Exception` instead of `RuntimeException`.